entrypoint.sh

```diff
@@ -33,6 +33,26 @@ cloud-api-adaptor-aws aws \
 	-socket /run/peerpod/hypervisor.sock
 }
 
+ibmcloud() {
```

Comment:
nit: should we test_vars here? also i just noticed it's disabled on aws

Reply:
I don't think it's needed. Will remove it.